src/dune/custom_build_info.ml

```diff
@@ -0,0 +1,16 @@
+open! Stdune
+
+type t =
+  { max_size : int
+  ; action : Loc.t * Action_dune_lang.t
+  }
+
+let output_file = "custom_build_info.txt-gen"
+
+let decode () =
+  let open Dune_lang.Decoder in
+  field_o "custom_build_info"
+    (fields
+       (let+ max_size = field "max_size" int
```

Comment:
I wonder if it would make sense to have a (documented and versioned) default that works out of the box?

Reply:
I am not sure to understand what you mean ?

(I do plan to add versioning using `Syntax.since`)

Reply:
Here, the user always has to provide `(max_size)`. I was suggesting that there could be a default so that when nothing is passed, there is an implicit `(max_size 40)` (or another value).